src/components/uikit/Text.tsx

```diff
@@ -1,19 +1,58 @@
-import { styled, themeProp, getTextTypographyVariants } from '~styles';
+import { Text as RNText } from 'react-native';
```

Comment:
We could use the "lean text" version of the `Text` component if we want to squeeze all the perf gains: [code](https://gist.github.com/hirbod/03d487f40b4c091d2c56ebfb17dba7ed), [tweet](https://x.com/FernandoTheRojo/status/1707762822015267219)

So maybe we use the lean text version as the default uikit text component and export an additional `InteractiveText` component that support all the functionality of the regular  RN `Text` component - such as `onPress` prop etc. Most of the time those "extra" props are not needed.